src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -142,6 +143,10 @@ public void apply() throws ConnectException {
       log.error("Error applying WAL file '{}' because it is corrupted: {}", logFile, e);
       log.warn("Truncating and skipping corrupt WAL file '{}'.", logFile);
       close();
+    } catch (CannotObtainBlockLengthException e) {
+      log.error("Error applying WAL file '{}'as not able to obtain block length: {}", logFile, e);
```

Comment:
Fixing a typo. But can we get a more specific (but still concise here) description of what this exception means?

```suggestion
      log.error("Error applying WAL file '{}' because the task cannot obtain ... ?: {}", logFile, e);
```